<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationDialog</class>
 <widget class="QDialog" name="SimulationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Simulation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0" colspan="3">
       <widget class="QComboBox" name="Simulatiion_comboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>A small container</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A large container</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>front of two small containers</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>after of two small containers</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Clear pictures</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="Capture_pushButton">
        <property name="text">
         <string>Capture</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="while_cycle_capture_checkBox">
        <property name="text">
         <string>Loop Capture Pictures</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>capture</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Lifting lever</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Fall rod</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Statistical</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>By counting</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>correct</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_9"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>error</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_10"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Recognition rate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_8"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2">
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="4" column="0" rowspan="2">
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>537</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
